### ISSUE TITLE
Intellisense suggestions is very slow with scikit learn package

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#16472]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.75.0
- OS Version: PopOs 22.04 LTS

Steps to Reproduce:
1. Clean installation of VSCode, with only Material Icon and the last Python extensions.
1. Miniconda environment with packages Scikit-learn 1.12, Pytorch 1.13.1
2. The following code causes slow suggests to intellisense (specific when I import scikit-learn package)
3. [https://gist.github.com/YachaywasiAI/1ac6b0511f828b487de4ef25f1eab0ea](url)
4. When I type last line, the parameters inside of function roc_auc_score. The time to suggest "labels" or "predictions" is high.
